serveradmin/serverdb/query_committer.py

```diff
@@ -616,6 +626,91 @@ def _validate_real_attributes(servertype, real_attributes):     # NOQA: C901
         violations_attribs,
     )
 
+def _resolve_servertype_changes(changed, joined_attributes):
+    """Expand servertype changes
+
+    Changing servertypes likely entails deleting attributes not available on
+    the target servertype.
+    """
+
+    # Filter for servertype changes
+    servertype_changes = [
+        change for change in changed
+        if 'servertype' in change.keys()
+    ]
+
+    # No servertype changes, we're done here
+    if not servertype_changes:
+        return changed
```

Comment:
Nobody checks the return value of this function.  You can just `return` to exit early.

Reply:
True.